hbase-server/src/main/java/org/apache/hadoop/hbase/replication/regionserver/HBaseInterClusterReplicationEndpoint.java

```diff
@@ -127,6 +127,7 @@
   private boolean dropOnDeletedTables;
   private boolean dropOnDeletedColumnFamilies;
   private boolean isSerial = false;
+  private long lastSinkFetchTime;
```

Comment:
JVM will initialize this to zero which is of consequence to the log message (will make sure that you get the `LOG.warn` the first time).

Explicitly initialize it here so with a comment so we know that?

Reply:
+1